daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/transaction/PartialTransaction.scala

```diff
@@ -69,27 +69,34 @@ private[lf] object PartialTransaction {
     */
   final case class Context(
       info: ContextInfo,
+      oldChild: Boolean, //one or more children predate exceptions
```

Comment:
How about tracking `minChildVersion` instead? I tend to prefer moving to a boolean only at the end and keep as much information in between.

Reply:
I had it this way to start with, but switched to the bool because it seemed tracking the precise version was more than required, and anyway a rollback node doesn't have a version as such... but you do know if you every created a rollback node then none of its children could be too old.

Reply:
But I'll change to tracking the version if you think this is clearer.

Reply:
done